src/components/Category/Category.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import './Category.css'
+
+type categoryProps = {
+  category: string;
+  categoryLength: number;
+  handleCatSelection(e:React.ChangeEvent<HTMLInputElement>):void; 
+}
```

Comment:
I haven't seen a prop declaration like this. This looks better than the arrow function.